### ISSUE TITLE
Note during R CMD check due to extract.oglmx

### ISSUE THREAD

**Issue by leifeld** [leifeld/texreg#120]
With the current commit 9d0d58684c0edbc4ad92d54e3bd744610d9f12cb, I get the following notes during `R CMD check`:

```
* checking R code for possible problems ... NOTE
extract.oglmx: no visible binding for global variable ‘results.oprob’
extract,oglmx: no visible binding for global variable ‘results.oprob’
Undefined global functions or variables:
  results.oprob
```
I think something is missing in the `extract.oglmx` extension. The problem is caused by the following line:
```
n <- length(results.oprob$modelframes$Z)
```
That object is not defined anywhere before.

**Comment by czucca** [comment]
Hi Philip, 
PR #115 is already fixing this.

**Comment by leifeld** [comment]
Fixed in #115.
